lib/vsc/utils/generaloption.py

```diff
@@ -682,12 +688,12 @@ def get_env_options(self):
                     self.log.debug("Environment variable %s is not set" % env_opt_name)
 
         if candidates:
-            msg="Found %s environment variable(s) that are similar but do not match valid option(s): %s"
+            msg = "Found %s environment variable(s) that are prefixed with %s but do not match valid option(s): %s"
+            tup = (len(candidates), self.envvar_prefix, ','.join(candidates))
             if self.error_env_options:
-                func=self.log.error
+                self.error_env_option_method(msg, *tup)
             else:
-                func=self.log.debug
-            func(msg, len(candidates), ",".join(candidates))
```

Comment:
what's wrong with the `func` ? at least you don't need one of those `tup`. and leave the default to `log.debug`.